apps/crawler/src/crawler.module.ts

```diff
@@ -0,0 +1,20 @@
+import typeormConfig from '@common/typeorm.config';
+// import { Collections } from '@entities/Collections';
```

Comment:
Why do you save this code in comments if you moved entities to config file?

Reply:
Just forgot to clean up. Fixed.